src/main/java/meeseeks/task/Deadline.java

```diff
@@ -0,0 +1,22 @@
+package meeseeks.task;
+
+import java.time.LocalDateTime;
+
+public class Deadline extends Task {
+    protected LocalDateTime by;
+
+    public Deadline(String description, LocalDateTime by) {
+        super(description);
+        this.by = by;
+    }
+
+    @Override
+    public String toString() {
+        return "[D]" + super.toString() + " (by: " + by + ")";
+    }
+
+    @Override
+    public String toFileFormat() {
```

Comment:
```suggestion
    public String toStorageString() {
```
The function name seems to imply that the task is converted into a file